libs/build-nss-desktop.sh

```diff
@@ -58,10 +58,10 @@ fi
 # https://github.com/mozilla/application-services/issues/962
 if [[ "${CROSS_COMPILE_TARGET}" =~ "darwin" ]]; then
   # Generated from nss-try@111b54aaa644978464bec98848ecba6f69d3f42e.
```

Comment:
Is this "Generated from" comment still relevant? And this is still pinned to a specific NSS version - do you think it's worth commenting here on what the process is for updating it? (I guess it's just "find the later version you want, update the task ID and get the SHA256 value (from where)?"